### DISCUSSION TITLE
slack channel?

### DISCUSSION THREAD

**Discussion by wolfspyre** [auroraresearchlab/netbox-dns#263] [General]
I created #netbox-dns in the NetDev Community slack, in case it might be of value to anyone....

not saying it SHOULD be used, I'll kill it if y'all don't like the idea of it existing... but thought it might be helpful

**Comment by peteeckel** [comment]
Hi @wolfspyre, nice idea but I don't use Slack and I definitely won't. The idea of it existing is not something I will pass judgement on, but judging from the activity here in the discussions it will probably be very lonely ...

**Comment by wolfspyre** [comment]
Is there a medium you **would** find beneficial for discussions?

GH feels like the wrong tool for discussions/questions. I figured that since netbox seems to encourage slack as the more ephemeral topical chat medium, it'd be helpful.... I'll nuke the channel so's it doesn't confuse peeps.

**Reply by peteeckel** [reply]
The question is what you want to achieve.

Given the fact that there are participants in different time zones, I don't think a more real-time medium would make much sense, so there's not too much wrong with GitHub discussions, and it does have the 'all in one place' advantage. Adding a second channel would probably further reduce the group of participants, which is already fairly small (I talked to people running the development of an open-source monitoring tool and they told me that almost nobody ever contributes - I wouldn't believe it, but they were right).

As for Slack, they have a very bad reputation about their handling of personal data, and their terms of use are ... well, I wouldn't touch them with a ten feet pole. I used to be a big fan of the Usenet, but those days are gone (yes, I am that old :-)). Actually I'd prefer decentralised tools for obvious reasons, so Matrix would probably be a good idea. But, again - real-time communication halfway across the earth doesn't make much sense to me, and to be honest I communicate on so many channels right now that adding another one is not very appealing.
